### PR TITLE
Make UI Consistent, Update Gradle, Update Spotify SDK, Fix Warnings

### LINKED ISSUE #81: Make UI consistent
- Some pages don't look the same 
- Drop downs are huge? In `EditRoutinesScreen` when selecting BPM type
- Consistency in capitalization and fonts
- Make dropdown taller in routines page

### PR DESCRIPTION
**Pull request by daltonbown01**
**Summary of Changes**

- Fixed bug where you needed to manually modify the manifest file
- Fixed warnings across the codebase
- Migrated to Material UI 3
- Updated Gradle
- Removed unused assets
- Update Spotify SDK
- Simplified Spotify dependencies by moving auth library out of `/libs`
- Made UI consistent
- Fixed dropdown consistency issues
- Fixed previews for various pages that couldn't render the previews

**UI Changes**
Screenshot 2023-07-23 at 12 51 12 AM
Screenshot 2023-07-23 at 12 51 22 AM
Screenshot 2023-07-23 at 12 51 29 AM
Screenshot 2023-07-23 at 12 53 05 AM
Screenshot 2023-07-23 at 12 53 10 AM

### CODE REVIEW

**Review by sjChen15 — APPROVED** [review]
I pulled the branch and did some testing, looks good!

File: app/src/main/java/com/example/musictoned/editRoutine/EditRoutineScreen.kt [hunk 1]
```diff
@@ -90,27 +88,26 @@ import java.util.Locale
...
     var popupControl by remember { mutableStateOf(false) }
 
-    var exercises = remember { mutableStateListOf<WorkoutExercise>() }
+    val exercises = remember { mutableStateListOf<WorkoutExercise>() }
 
-    var swapped = remember { mutableStateOf(false) }
+    val swapped = remember { mutableStateOf(false) }
 
-    exercises.addAll(workout.exercises.toList())
+    exercises.addAll(workout.component1().exercises.toList())
```
Comment (FlightlessGoose) [inline app/src/main/java/com/example/musictoned/editRoutine/EditRoutineScreen.kt:110]: what is the purpose of component1()?

**Review by FlightlessGoose — APPROVED** [review]
I reviewed the files in editRoutine/, routine/, routines/, ui/theme/, spotify/, and workoutcreation/

All these changes make sense, but I just had one question for more own understanding.

On EditRoutineScreen.kt, you changes this line:

exercises.addAll(workout.exercises.toList())

to

exercises.addAll(workout.component1().exercises.toList())

What is the purpose of component1()?


Aside from that, I will give my approval to these files specifically

File: app/src/main/java/com/example/musictoned/editRoutine/EditRoutineScreen.kt (hunk 1 above)
Comment (daltonbown01) [reply app/src/main/java/com/example/musictoned/editRoutine/EditRoutineScreen.kt:110]: The `workout` object changed to a `MutableState<Exercise>` object, so in order to touch the contained object you need to call `.componentX()`

Comment (daltonbown01) [reply app/src/main/java/com/example/musictoned/editRoutine/EditRoutineScreen.kt:110]: See line 93

**Comment by daltonbown01** [comment]
> I reviewed the files in editRoutine/, routine/, routines/, ui/theme/, spotify/, and workoutcreation/
> 
> All these changes make sense, but I just had one question for more own understanding.
> 
> On EditRoutineScreen.kt, you changes this line:
> 
> exercises.addAll(workout.exercises.toList())
> 
> to
> 
> exercises.addAll(workout.component1().exercises.toList())
> 
> What is the purpose of component1()?
> 
> Aside from that, I will give my approval to these files specifically

The `workout` object changed to a `MutableState<Exercise>` object, so in order to touch the contained object you need to call `.componentX()`. See line 93. It was a change to fix the preview not working

**Pull request merged**